appveyor.yml

```diff
@@ -26,17 +26,44 @@ before_build:
 build:
   project: Flow.Launcher.sln
   verbosity: minimal
+after_build:
+  - ps: .\Scripts\post_build.ps1
 
 artifacts:
-- path: 'Output\Packages\Flow-Launcher-*.zip'
-  name: Zip
 - path: 'Output\Release\Flow.Launcher.Plugin.*.nupkg'
   name: Plugin nupkg
+- path: 'Output\Packages\Flow-Launcher-*.exe'
+  name: Squirrel Installer
+- path: 'Output\Packages\FlowLauncher-*-full.nupkg'
+  name: Squirrel nupkg
+- path: 'Output\Packages\RELEASES'
+  name: Squirrel RELEASES
 
 deploy:
-  provider: NuGet
-  artifact: /.*\.nupkg/
-  api_key:
-    secure: n80IeWR3pN81p0w4uXq4mO0TdTXoJSHHFL+yTB9YBJ0Wni2DjZGYwOFdaWzW4hRi
-  on:
-    branch: master
\ No newline at end of file
+  - provider: NuGet
+    artifact: Plugin nupkg
+    api_key:
+      secure: n80IeWR3pN81p0w4uXq4mO0TdTXoJSHHFL+yTB9YBJ0Wni2DjZGYwOFdaWzW4hRi
+    on:
+      branch: master
+
+  - provider: GitHub
+    release: v$(flowVersion)
+    auth_token:
+      secure: ij4UeXUYQBDJxn2YRAAhUOjklOGVKDB87Hn5J8tKIzj13yatoI7sLM666QDQFEgv
+    artifact: Squirrel Installer, Squirrel nupkg, Squirrel RELEASES
+    draft: true
+    force_update: true
+    on:
+      branch: master
+      APPVEYOR_REPO_TAG: false
+
+  - provider: GitHub
+    release: v$(flowVersion)
+    auth_token:
+      secure: ij4UeXUYQBDJxn2YRAAhUOjklOGVKDB87Hn5J8tKIzj13yatoI7sLM666QDQFEgv
+    artifact: Squirrel Installer, Squirrel nupkg, Squirrel RELEASES
+    force_update: true
+    on:
+      branch: master
+      APPVEYOR_REPO_TAG: true
```

Comment:
is this correct? apparently the tag eval is not going to work if you combine it with branch:
![image](https://user-images.githubusercontent.com/26427004/103723931-48303e80-5027-11eb-83e1-9f3f3ab46b6e.png)

Reply:
@jjw24 deploy conditions are not finalized, see my other comment